tests/jvm/src/test/scala/scala/meta/tests/metacp/BaseMetacpSuite.scala

```diff
@@ -35,6 +35,7 @@ abstract class BaseMetacpSuite extends BaseCliSuite {
   val kafka = Library("org.apache.kafka", "kafka_2.12", "1.0.0")
   val flink = Library("org.apache.flink", "flink-parent", "1.4.1")
   val grpc = Library("io.grpc", "grpc-all", "1.10.0")
+  val play = Library("com.typesafe.play.", "play_2.12", "2.6.12")
```

Comment:
By the way, I've digged a bit into the code and couldn't figure out why it works anyway. I suspected `coursier.Module` or `coursier.Dependency` would do some kind of input sanitization, but that wasn't the case.

It could be some maven trick I'm not aware of, but so far it remains a quirky behavior to me.